torchvision/transforms/transforms.py

```diff
@@ -1578,7 +1578,10 @@ def get_params(
 
         for _ in range(10):
             erase_area = area * torch.empty(1).uniform_(scale[0], scale[1]).item()
-            aspect_ratio = torch.empty(1).uniform_(ratio[0], ratio[1]).item()
+            log_ratio = torch.log(torch.tensor(ratio))
```

Comment:
The `log_ratio` estimation could move outside of the loop as does not need to be re-estimated every time.

Reply:
LGTM!

Reply:
Cool, thanks for changing it on the other place as well.